website/docs/user_guide/technical-debt.md

```diff
@@ -15,8 +15,8 @@ A toggle being (potentially) stale, does not affect how it performs in your appl
 
 
 ## The technical debt dashboard
-
-In order to assist with removing unused feature toggles, Unleash provides a technical debt dashboard in the management-ui. You can find it by clicking on “Advanced” in the top-line menu then choose _Reporting_ in the  dropdown menu.
+@Thomas - is this technical debt dashboard still the correct naming? (wouldn't make more sens to updat it Health? in the entire page)
```

Comment:
How about a compromise? Change the title to "Technical debt and project health"? Or maybe just "project health"? 🤔